.github/workflows/build-bottles.yml

```diff
@@ -18,33 +18,37 @@ jobs:
       HOMEBREW_DEVELOPER: 1
       HOMEBREW_NO_ANALYTICS: 1
       HOMEBREW_NO_AUTO_UPDATE: 1
+      GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED: 1
     steps:
-      - name: Setup tap
+      - name: Checkout repository
         run: |
-          git -C $(brew --repo) remote set-url origin https://github.com/Homebrew/brew
-          cd $(brew --repo ${{ github.repository }})
-          git config remote.origin.url https://github.com/Homebrew/homebrew-core/
-          git fetch origin refs/pull/${{ github.event.pull_request.number }}/merge
-          git checkout -f FETCH_HEAD
-      - name: Cleanup
+          cd $(brew --repo ${{github.repository}})
+          git fetch origin ${{github.sha}}
+          git reset --hard ${{github.sha}}
+      - name: Customize brew-test-bot
```

Comment:
This can be dropped once the PR is merged, right?

Reply:
Yes: https://github.com/Homebrew/homebrew-core/pull/50620/commits/bfbbab9f4d0cc1fb9405dd3f0a2376533f061eef